### ISSUE TITLE
Installing plugin vagrant-libvirt fails on ubuntu 18.04

### ISSUE THREAD

**Issue by ingobecker** [sous-chefs/vagrant#101]
### Cookbook version
0.9.1

### Chef-client version
14.10.9

### Platform Details
ubuntu 18.04.2

### Scenario:
Installation of the vagrant-libvirt plugin like described in #100 fails (see output below). Installing it as the actual user running the same command as used by the ` shell_out` works as expected.

```
[2019-03-22T11:20:22+00:00] ERROR: Mixlib::ShellOut::ShellCommandFailed: vagrant_plugin[vagrant-libvirt] (vagrant::install_plugins line 26) had an error: Mixlib::ShellOut::ShellCommandFailed: Expected process to exit with [0], but received '1'
---- Begin output of vagrant plugin install vagrant-libvirt ----
STDOUT: Installing the 'vagrant-libvirt' plugin. This can take a few minutes...
Building native extensions.  This could take a while...
Building native extensions.  This could take a while...
STDERR: Bundler, the underlying system Vagrant uses to install plugins,
reported an error. The error is shown below. These errors are usually
caused by misconfigured plugin installations or transient network
issues. The error from Bundler is:

ERROR: Failed to build gem native extension.

    current directory: /home/ingo/.vagrant.d/gems/2.4.4/gems/ruby-libvirt-0.7.1/ext/libvirt
/opt/vagrant/embedded/bin/ruby -r ./siteconf20190322-24410-h69yh1.rb extconf.rb
*** extconf.rb failed ***
Could not create Makefile due to some reason, probably lack of necessary
libraries and/or headers.  Check the mkmf.log file for more details.  You may
need configuration options.

Provided configuration options:
        --with-opt-dir
        --with-opt-include
        --without-opt-include=${opt-dir}/include
        --with-opt-lib
        --without-opt-lib=${opt-dir}/lib
        --with-make-prog
        --without-make-prog
        --srcdir=.
        --curdir
        --ruby=/opt/vagrant/embedded/bin/$(RUBY_BASE_NAME)
        --with-libvirt-include
        --without-libvirt-include
        --with-libvirt-lib
        --without-libvirt-lib
        --with-libvirt-config
        --without-libvirt-config
        --with-pkg-config
        --without-pkg-config
extconf.rb:73:in `<main>': libvirt library not found in default locations (RuntimeError)

To see why this extension failed to compile, please check the mkmf.log which can be found here:

  /home/ingo/.vagrant.d/gems/2.4.4/extensions/x86_64-linux/2.4.0/ruby-libvirt-0.7.1/mkmf.log

extconf failed, exit code 1

Gem files will remain installed in /home/ingo/.vagrant.d/gems/2.4.4/gems/ruby-libvirt-0.7.1 for inspection.
Results logged to /home/ingo/.vagrant.d/gems/2.4.4/extensions/x86_64-linux/2.4.0/ruby-libvirt-0.7.1/gem_make.out
---- End output of vagrant plugin install vagrant-libvirt ----
Ran vagrant plugin install vagrant-libvirt returned 1

```

**Comment by MarkGibbons** [comment]
Hello,
  Could you create a test case that fails?  
thanks,
  Mark

**Comment by MarkGibbons** [comment]
Was able to reproduce the error.
vagrant-libvirt/vagrant-libvirt#891 seems relevant. The install works when the config options are specified. Trying to figure out how to fit those options into the cookbook.

**Comment by MarkGibbons** [comment]
I'll have a PR soon that let's you specify any needed environment variables needed for installing a specific plugin.

**Comment by MarkGibbons** [comment]
Merged #103.  I'll ship it to supermarket.chef.io soon.
